src/components/TodoList/TodoList.jsx

```diff
@@ -1,4 +1,9 @@
-// Add the required props
-export const TodoList = () => (
-  <>TodoList markup</>
+import { TodoInfo } from '../TodoInfo';
+
+export const TodoList = ({ todos, users }) => (
```

Comment:
The `users` prop is defined but not used in the component. If user information should be displayed according to the task, ensure you pass user data to the `TodoInfo` component, or remove the unused `users` prop if it's not necessary.